nixos/modules/services/web-apps/hedgedoc.nix

```diff
@@ -1,1001 +1,254 @@
 { config, lib, pkgs, ... }:
-
-with lib;
-
 let
+  inherit (lib) versionAtLeast mkEnableOption mkOption types mdDoc mkIf
+    optionals mkPackageOption mkRemovedOptionModule mkDefault
+    mkRenamedOptionModule literalExpression maintainers;
```

Comment:
I am not keen on maintaing this list

Reply:
I was trying to avoid using `with lib;`. I'll go ahead and prefix the functions with `lib.`

Reply:
I prefixed all which occured less than 5 times. It became stupid trying to prefix all `mkOption`s and stuff.